CF/routing-readme.md

```diff
@@ -0,0 +1,13 @@
+## Routing
+
+Once the cdn service is set up and the apps have been deployed we need to set up
+some final routing.
+
+```
+cf target -s staging
+cf map-route ccs-rmi-app-staging --hostname www staging.rmi-paas.dxw.net
```

Comment:
When running this for sandbox we found that the `bind-route-service` command will fail with `api not found` unless the route is mapped before hand:

```
cf map-route ccs-rmi-api-staging --hostname api staging.rmi-paas.dxw.net
```

Same for production further down.